docker/postgres-techempower-fortunesbench/run.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+if (( $# != 1 ))
+then
+    echo "Usage: run.sh server-name-or-ip"
+    exit 1
+fi
+
+host=$1
+threads=$(nproc --all)
+connections=$(($threads * 8))
+
+#echo on
+set -x
+
+docker run \
+    -it \
```

Comment:
The `postgres-techempower-fortunesbench` is a client app which runs a raw SQL benchmark against postges, simulating the queries run by the `fortunes` web benchmark.  It's designed to run in a container, print the benchmark results, and exit.